architecture/additional_concepts/flannel.adoc

```diff
@@ -29,4 +31,9 @@ responsibile for:
 
 Each *flanneld* agent provides this infomation to a centralized *etcd* store so
 other agents on hosts can create an overlay network and route packets to
-any container within the *flannel* network.
\ No newline at end of file
+any container within the *flannel* network.
+
+The following diagram illustrates the architecture and data flow from one
+container to another using a *flannel* network:
+
+image::flannel.png[Flannel Communication]
```

Comment:
@tracyrankin - are you able to help here? Just need someone to review the diagram. Thanks!